scripts/bump_version.py

```
@@ -18,21 +19,236 @@
+        new.trivia.comment_ws = existing.trivia.comment_ws
+        new.trivia.comment = existing.trivia.comment
+        new.trivia.trail = existing.trivia.trail
+        entry["version"] = new
+    else:
+        entry["version"] = prefix + version
+
+
+def _update_string_dependency(
+    deps: MutableMapping[str, object],
+    dependency: str,
+    entry: Any,
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `entry`

(ANN401)